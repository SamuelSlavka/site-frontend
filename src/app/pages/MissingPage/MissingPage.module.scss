.MissingPage {
  @apply min-h-full h-fit;
}