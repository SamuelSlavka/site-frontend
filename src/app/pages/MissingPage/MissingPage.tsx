import React, { FC } from "react";
import styles from "./MissingPage.module.scss";

interface MissingPageProps {}

const MissingPage: FC<MissingPageProps> = () => (
    <div className={styles.MissingPage} data-testid="MissingPage">
        <h1 className="text-center text-5xl font-bold py-16">Wrong turn</h1>
    </div>
);

export default MissingPage;
