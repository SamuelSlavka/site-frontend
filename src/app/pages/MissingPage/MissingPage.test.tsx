import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import MissingPage from "./MissingPage";
import renderer from 'react-test-renderer';
import React from "react";

describe("MissingPage works", () => {
    test("it should mount", () => {
        render(<MissingPage />);

        const missingPage = screen.getByTestId("MissingPage");
        expect(missingPage).toBeInTheDocument();
    });

    test("it should render correnctly", () => {
        const component = renderer.create(
            <MissingPage />
        );

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
