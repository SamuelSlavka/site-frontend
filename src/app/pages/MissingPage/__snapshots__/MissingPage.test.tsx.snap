// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissingPage works it should render correnctly 1`] = `
<div
  className="MissingPage"
  data-testid="MissingPage"
>
  <h1
    className="text-center text-5xl font-bold py-16"
  >
    Wrong turn
  </h1>
</div>
`;
